controllers/main.js

```diff
@@ -150,13 +162,6 @@ module.exports.renderDetailedResults = async (req, res, next) => {
   next();
 };
 
-// module.exports.renderTest = async (req, res) => {
-//   const image = await getPixelatedImage(req.gameData.background_image);
-
-//   res.render("main/test.ejs", {
-//     image,
-//     gameName: req.gameData.name,
-//     imgLink: req.gameData.background_image,
-//     extraStyles: cardStyle,
-//   });
-// };
+module.exports.renderTest = async (req, res) => {
```

Comment:
What does this do? Seems like an empty page with word "TEST" in it

Reply:
As i had mentioned at some point a while ago i prefer to use to use a `/test` page to check out some of the new stuff that i had implemented. I used this to test if the middleware that ive been writing was working properly